<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-LPT61RQN" failures="1" tests="5" name="Package with subpackages" time="33.305" errors="0" timestamp="2021-06-08T00:40:01 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.638"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.223">
    <failure type="junit.framework.AssertionFailedError" message="Test fail">
      <![CDATA[junit.framework.AssertionFailedError: Test fail
at junit.framework.Assert.fail(Assert.java:57)
at stepDef.steps.user_should_able_to_export_the_report(steps.java:77)
at âœ½.User should able to export the report(file:///C:/Users/anjan/eclipse-workspace/CucumberProject/src/test/java/features/feature2.feature:34)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.393"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.256"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.185"/>
</testsuite> <!-- Package with subpackages -->
